--- a/JBBCode/Parser.php
+++ b/JBBCode/Parser.php
@@ -60,6 +60,6 @@
      * @return Parser
      */
     public function addBBCode($tagName, $replace, $useOption = false, $parseContent = true, $nestLimit = -1,
-                              InputValidator $optionValidator = null, InputValidator $bodyValidator = null)
+                              ?InputValidator $optionValidator = null, ?InputValidator $bodyValidator = null)
     {
         $builder = new CodeDefinitionBuilder($tagName, $replace);
